troubadix/standalone_plugins/deprecate_vts.py

```diff
@@ -0,0 +1,249 @@
+# SPDX-License-Identifier: GPL-3.0-or-later
+# SPDX-FileCopyrightText: 2024 Greenbone AG
+
+import re
+from argparse import ArgumentParser, Namespace
+from dataclasses import dataclass
+from pathlib import Path
+from typing import Iterable, Optional
+
+from troubadix.argparser import file_type, directory_type
+from troubadix.helper.patterns import (
+    get_special_script_tag_pattern,
+    get_script_tag_pattern,
+    ScriptTag,
+    SpecialScriptTag,
+)
+
+DEPRECATIONS = {
+    "notus": "and replaced by a Notus scanner based one.",
+    "merged": "because it has been merged into a different VT.",
+    "defunct": "and is therefore no longer functional.",
+    "duplicate": "as a duplicate.",
+}
```

Comment:
If you have an idea off the top of your head how to use an Enum instead, I'd prefer that. IIRC Enums and argparse are a bit nasty, so feel free to resolve this, if it can't be implemented quickly.